bindings/c/test/fdb_c_client_config_tests.py

```diff
@@ -568,14 +568,14 @@ def test_external_client(self):
         test.api_version = api_version_from_str(PREV_RELEASE_VERSION)
         test.exec()
         test.check_initialization_state("created")
-        test.check_healthy_status(False)
+        test.check_healthy_status(True)
         test.check_available_clients([PREV_RELEASE_VERSION, CURRENT_VERSION])
         test.check_current_client(PREV_RELEASE_VERSION)
 
     def test_external_client_unsupported_api(self):
         # Leaving an unsupported API version
         test = ClientConfigTest(self)
-        test.create_external_lib_path(PREV_RELEASE_VERSION)
+        test.create_external_lib_path(PREV2_RELEASE_VERSION) # Require 7.1 for the test to work
         test.expected_error = 2204  # API function missing
```

Comment:
Hi @sfc-gh-vgasiunas , for this test and other tests below, in order to produce `api_function_missing`, we need to use 7.1 or earlier version. I used `PREV2_RELEASE_VERSION` now but not sure how would we change it once we want to update FDB to 7.5.

Reply:
The idea of the test is that setting API to CURRENT_VERSION while using a client of PREV_RELEASE_VERSION results in an error. However, I am not sure what happens if there are no new functions added to the current version in the API. Do you get a success or some other error when executing it with PREV_RELEASE_VERSION?

Reply:
When using PREV_RELEASE_VERSION, instead of getting API function missing, I'm getting 2203 (api_version_not_supported), which makes sense when using --api-version 740 on a 7.3 client.

Reply:
Then just update the expected error to 2203

Reply:
Make sense. Done.